nms/app/packages/magmalte/app/components/context/RefreshContext.js

```diff
@@ -35,13 +42,24 @@ type Props = {
   lastRefreshTime?: string,
 };
 
-type refreshType = 'subscriber' | 'gateway' | 'enodeb';
+type refreshType = 'subscriber' | 'gateway' | 'feg_gateway' | 'enodeb';
```

Comment:
String union types are a little strange to use. If somebody needs to update one of the values they would need to find and replace all occurrences.

```suggestion
const RefreshTypeEnum = Object.freeze({SUBSCRIBER: 'subscriber', GATEWAY: 'gateway', ...})
type RefreshType = $Values<typeof RefreshTypeEnum>;
```

https://stackoverflow.com/questions/287903/how-can-i-guarantee-that-my-enums-definition-doesnt-change-in-javascript
https://flow.org/en/docs/types/utilities/

Reply:
Where there's comparisons to `'feg_gateway'`, they can then be updated as well
```
if (props.type === RefreshTypeEnum.FEG_GATWAY) {
```